### PR TITLE
LRQA-44184 | master

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LRQA-44184

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by weller0246** [comment]
Conflict is resolved.
@gaborlovasdotcom

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
@weller0246 I fixed the SF issue about a missing char in LDAP.macro

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
@weller0246 Could you please rebase?

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Review by gaborlovasdotcom — APPROVED** [review]
Looks good, let's run SF

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro [hunk 1]
```diff
@@ -93,6 +125,14 @@ definition {
 		UserGroup.viewRemoveMemberCP(
 			userFirstName = "${userFirstName}",
 			userGroupName = "${userGroupName}");
+
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro:128]: @weller0246 Closing brackets are missing from this macro.

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Please fix the missing bracket.

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**